static/sass/patterns/_forms.scss

```diff
@@ -25,25 +25,3 @@
 .form-actions {
   text-align: right;
 }
-
-.btn {
-  &,
-  #{$link} {
-    @include contrasted('action');
-    background-image: none; // remove link underline when styling as button...
-    border-color: transparent;
-    border-style: solid;
-    border-width: 1px;
-    display: inline-block;
-    padding: 0.5rem 1rem;
-    text-decoration: none;
-    text-shadow: none;
-    transition: color 150ms, background-color 150ms;
```

Comment:
Yeah, looks like a number of the additions just merged in via #69 were lost in the merge conflicts: https://github.com/oddbird/oddsite/pull/69/files#diff-846d4cdada09262ed70b32522b66b16f